<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>منو</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="creates_farm">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>ایجاد دامداری</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="creates_market">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>ورود به فروشگاه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="farm_list">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>لیست دامداری ها</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sell_products">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>فروش محصولات</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deposit_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>افزایش موجودی</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="money_icon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>موجودی :</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Timer</string>
    </property>
    <widget class="QTextEdit" name="minutesText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="secondsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="hundredthsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startStopButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
